src/test/java/com/jwplayer/southpaw/util/ByteArraySetTest.java

```diff
@@ -46,6 +74,655 @@ public ByteArraySet createSmallSet() {
         return set;
     }
 
+    public List<ByteArray> getRandomByteArrays(int count, long seed) {
+        System.out.println("Generating random " + count + " Byte Array(s) with " + seed + " seed");
+        Random randomSeed = new Random(seed);
+        List<ByteArray> vals = Stream.generate(() -> RandomStringUtils.random(RANDOM_STRING_SIZE, 0, 0, true, true, null, randomSeed))
+                .limit(count)
+                .collect(Collectors.toSet())
+                .stream()
+                .map(str -> new ByteArray(str.getBytes()))
+                .collect(Collectors.toList());
+        assertEquals(count, vals.size());
+        return vals;
+    }
+
+    public List<ByteArray> getRandomByteArrays(int count) {
```

Comment:
These `getRandomByteArrays` and `getRandomByteArrays` function definitions can be combined for simplicity. They essentially have the same signature now

Reply:
We use both this functions, I'm not sure we can combine them.

Reply:
Also whichever one remains can be changed to `private` function